packages/x-charts-pro/src/context/ZoomProvider/ZoomContext.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { Initializable } from '@mui/x-charts/internals';
+
+export type ZoomState = {
+  zoomRange: [number, number];
+  setZoomRange: (range: [number, number]) => void;
+  isInteracting: boolean;
+  setIsInteracting: (isInteracting: boolean) => void;
+};
+
+export const ZoomContext = React.createContext<Initializable<ZoomState>>({
+  isInitialized: false,
+  data: {
+    zoomRange: [0, 100],
+    setZoomRange: () => {},
+    isInteracting: false,
+    setIsInteracting: () => {},
```

Comment:
Nope :)

I was going for making sure the functionality works first, then I could shape the API, but I thought we agreed on improving that on a future PR. 😅

Reply:
> making sure the functionality works first, then I could shape the API

Yes for the coding part, but it would be great to have at least an idea about how you would like to do it. Just to be sure we have a path to it and don't end up with something that can only support one zoom in one direction